compliance/node/index/indexer.go

```diff
@@ -108,13 +114,20 @@ type NodeIndexerConfig struct {
 	Timeout time.Duration
 }
 
-// DefaultNodeIndexerConfig is the default configuration for a node indexer.
-var DefaultNodeIndexerConfig = NodeIndexerConfig{
-	HostPath: env.NodeIndexHostPath.Setting(),
-	// The default, mTLS-capable client will be used.
-	Client:             nil,
-	Repo2CPEMappingURL: env.NodeIndexMappingURL.Setting(),
-	Timeout:            10 * time.Second,
+// DefaultNodeIndexerConfig provides the default configuration for a node indexer.
+func DefaultNodeIndexerConfig() NodeIndexerConfig {
+	return NodeIndexerConfig{
+		HostPath: env.NodeIndexHostPath.Setting(),
+		// The default, mTLS-capable client will be used.
+		Client:             nil,
+		Repo2CPEMappingURL: buildMappingURL(),
+		Timeout:            10 * time.Second,
+	}
+}
+
+func buildMappingURL() string {
+	URL := env.AdvertisedEndpoint.Setting() + scannerDefinitionsRouteInSensor + "?file=" + sensorMappingsFile
```

Comment:
Is `NodeIndexMappingURLPath` used?

Reply:
Done in https://github.com/stackrox/stackrox/pull/14766/commits/9a63299fc4e5fd1ce0b79394791fd68f5d8ec21b